src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1260,26 +1263,42 @@ namespace SurfaceGeometry {
         } // ...end of the Surface DO loop for finding BaseSurf
         //**********************************************************************************
 
-        // The surfaces need to be hierarchical.  Input is allowed to be in any order.  In
-        // this section it is reordered into:
+        // The surfaces need to be hierarchical by zone.  Input is allowed to be in any order.  In
+        // this section the surfaces are reordered into:
```

Comment:
Here's where the surface re-ordering happens. This is where future padding would happen.